hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java

```diff
@@ -1106,22 +1108,24 @@ public MiniHBaseCluster startMiniHBaseCluster(StartMiniClusterOption option)
       conf.setInt(ServerManager.WAIT_ON_REGIONSERVERS_MAXTOSTART, option.getNumRegionServers());
     }
 
+    // Avoid log flooded with chore execution time, see HBASE-24646 for more details.
+    Log4jUtils.setLogLevel(org.apache.hadoop.hbase.ScheduledChore.class.getName(), "INFO");
+
     Configuration c = new Configuration(this.conf);
     TraceUtil.initTracer(c);
-    this.hbaseCluster =
-        new MiniHBaseCluster(c, option.getNumMasters(), option.getNumAlwaysStandByMasters(),
-            option.getNumRegionServers(), option.getRsPorts(), option.getMasterClass(),
-            option.getRsClass());
+    this.hbaseCluster = new MiniHBaseCluster(c, option.getNumMasters(),
+      option.getNumAlwaysStandByMasters(), option.getNumRegionServers(), option.getRsPorts(),
+      option.getMasterClass(), option.getRsClass());
     // Populate the master address configuration from mini cluster configuration.
     conf.set(HConstants.MASTER_ADDRS_KEY, MasterRegistry.getMasterAddr(c));
     // Don't leave here till we've done a successful scan of the hbase:meta
-    Table t = getConnection().getTable(TableName.META_TABLE_NAME);
-    ResultScanner s = t.getScanner(new Scan());
-    while (s.next() != null) {
-      continue;
+    try (Table t = getConnection().getTable(TableName.META_TABLE_NAME);
+      ResultScanner s = t.getScanner(new Scan())) {
+      while (s.next() != null) {
+        continue;
```

Comment:
`continue` is redundant, we can remove it and have just one comment stating this is empty loop. But this is not a strong opinion, only if you feel worth doing it :)

Reply:
Use a for(;;) to implement the logic. I'm afraid an empty {} will lead to a checkstyle warning.

Reply:
Agree, +1

Reply:
I like 'continue' instead of empty loop.

Reply:
... makes the intent clear.